GnuWin32.Tree version 1.5.2.2

Dragon1573 opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37973)

Dragon1573 committed:
GnuWin32.Tree version 1.5.2.2

ItzLevvie commented on manifests/g/GnuWin32/Tree/1.5.2.2/GnuWin32.Tree.locale.en-US.yaml:
```diff
@@ -0,0 +1,13 @@
+PackageVersion: 1.5.2.2
+PackageLocale: en-US
+Publisher: GnuWin32 <gnuwin32.sourceforge.net>
+PackageName: Tree
```
```suggestion
PackageName: "GnuWin32: Tree"
```

ItzLevvie replied:
This needs to match Control Panel.

ItzLevvie commented on manifests/g/GnuWin32/Tree/1.5.2.2/GnuWin32.Tree.locale.en-US.yaml:
```diff
@@ -0,0 +1,13 @@
+PackageIdentifier: GnuWin32.Tree
+PackageVersion: 1.5.2.2
+PackageLocale: en-US
+Publisher: GnuWin32 <gnuwin32.sourceforge.net>
```
```suggestion
Publisher: GnuWin32
```
This needs to match Control Panel.

ItzLevvie commented on manifests/g/GnuWin32/Tree/1.5.2.2/GnuWin32.Tree.installer.yaml:
```diff
@@ -0,0 +1,19 @@
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: http://downloads.sourceforge.net/gnuwin32/tree-1.5.2.2-setup.exe
```
```suggestion
  InstallerUrl: https://sourceforge.net/projects/gnuwin32/files/tree/1.5.2.2/tree-1.5.2.2-setup.exe/download
```

ItzLevvie commented on manifests/g/GnuWin32/Tree/1.5.2.2/GnuWin32.Tree.installer.yaml:
```diff
@@ -0,0 +1,19 @@
+PackageVersion: 1.5.2.2
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10
```
```suggestion
MinimumOSVersion: 10.0.0.0
```

Dragon1573 committed:
Update "PackageName" for Control Panel

The package name should be the same as in Control Panel

Co-authored-by: Levvie - she/her <11600822+ItzLevvie@users.noreply.github.com>

Dragon1573 committed:
Update "Publisher" for Control Panel

The Publisher information should be the same as in Control Panel.

Co-authored-by: Levvie - she/her <11600822+ItzLevvie@users.noreply.github.com>

Dragon1573 committed:
Update "MinimumOSVersion"

Windows is using "4-part version code". It should be complete.

Co-authored-by: Levvie - she/her <11600822+ItzLevvie@users.noreply.github.com>

Dragon1573 committed:
Update "InstallerUrl"

The previous URL for installer is copied from the offical homepage of "Tree for Windows". It should be changed to use the "direct link" from SourceForge.

Co-authored-by: Levvie - she/her <11600822+ItzLevvie@users.noreply.github.com>
